ui/app/templates/components/transit-key-action/encrypt.hbs

```diff
@@ -27,7 +25,13 @@
       </div>
     </div>
     <div class="field">
-      <Input @type="checkbox" id="encodedBase64" @checked={{@encodedBase64}} data-test-transit-input="encodedBase64" />
+      <Input
+        @type="checkbox"
+        id="encodedBase64"
+        @checked={{@encodedBase64}}
+        {{on "change" @toggleEncodeBase64}}
```

Comment:
You'll see the added property `toggleEncodeBase64`. I tried a mut operation but got the following error and couldn't seem to move past it, so opted to pass an action down, mutating the data in the parent. 
`Error: Assertion Failed: You can only pass a path to mut`

Tried versions of this 
`{{on "change" (fn (mut @encodeBase64) (not t@encodeBase64))}}`